project/frontend/src/FindMatch/FindMatch.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class PreferenceForm extends React.Component {
+    render() {
+        <div id='restaurant-prefs'>
+            <label for='cuisine'>What cuisine?</label>
+            <select name="cuisine" id="cuisine">
+                <option value='Italian'>Italian</option>
+                <option value='Mexican'>Mexican</option>
+                <option value='Indian'>Indian</option>
```

Comment:
For all of the option elements throughout the form, you can create arrays that contain all the options and then map through them (so you don't have to rewrite each time). 
For example:
`const cuisines = ['Italian', 'Mexican', 'Indian']`
then inside the `<select>` tag you can do `cuisines.map(cuisine => <option value={cuisine}>{cuisine}</option>`

See React List and Keys for more on this: https://reactjs.org/docs/lists-and-keys.html